scripts/workers/submission_worker.py

```diff
@@ -48,7 +48,7 @@
                                LeaderboardData) # noqa
 
 from jobs.models import Submission          # noqa
-
+from settings.common import RABBITMQ_PARAMETERS        # noqa
```

Comment:
Can you try `from django.conf import settings` and see if it works?

To refer `RABBITMQ_PARAMETERS`, you should use `settings.RABBITMQ_PARAMETERS`. Let me know if it does not works.